prime-router/src/main/kotlin/cli/main.kt

```diff
@@ -80,8 +112,7 @@ class RouterCli : CliktCommand(
         if (listOfFiles.isNullOrEmpty()) error("No files to merge.")
         val files = listOfFiles.split(",", " ").filter { ! it.isNullOrBlank() }
         if (files.isEmpty()) error("No files to merge found in comma separated list.  Need at least one file.")
-        val reports = files.map { readReportFromFile(metadata, it) }.filter { report -> report != null }
-        if (reports.isEmpty()) error("Unable to merge.  All reports failed validation")
+        val reports = files.map { readReportFromFile(metadata, it) }
         echo("Merging ${reports.size} reports.")
```

Comment:
@jimduff-usds the compiler was warning that filtering and empty checking here was redundant (since `readReportFromFile()` throws if `report == null`. I did this to clean up the error, but not sure if you would have rather put a try...catch in the map block so one bad file doesn’t stop the process.